pages/resources/message.mdx

```diff
@@ -1301,3 +1304,210 @@ Returns embed data from a list of URLs.
+
+If `answer_counts` does not contain an entry for a particular answer, then there are no votes for that answer.
+
+###### Poll Results Structure
+
+| Field         | Type                                                            | Description                                   |
+| ------------- | --------------------------------------------------------------- | --------------------------------------------- |
+| is_finalized  | boolean                                                         | Whether the votes have been precisely counted |
+| answer_counts | array[[poll answer count](#poll-answer-count-structure) object] | The counts for each answer                    |
+
+###### Poll Answer Count Structure
+
+| Field    | Type    | Description                                    |
+| -------- | ------- | ---------------------------------------------- |
+| id       | integer | The ID of the answer                           |
+| count    | integer | The number of votes for this answer            |
+| me_voted | boolean | Whether the current user voted for this answer |
+
+<RouteHeader method="GET" url="/channels/{channel.id}/polls/{message.id}/answers/{answer.id}">
+  Get Answer Voters
+</RouteHeader>
+
+Get a list of users that voted for this specific answer.
+
+###### Query String Params
+
+| Field  | Type      | Description                                       |
+| ------ | --------- | ------------------------------------------------- |
+| after? | snowflake | Get users after this user ID                      |
+| limit? | integer   | Max number of users to return (1-100, default 25) |
+
+###### Response Body
+
+| Field | Type                                                      | Description                     |
+| ----- | --------------------------------------------------------- | ------------------------------- |
+| users | array[partial [user](/resources/user#user-object) object] | Users who voted for this answer |
+
+<RouteHeader method="POST" url="/channels/{channel.id}/polls/{message.id}/expire">
+  End Poll
+</RouteHeader>
+
+Immediately ends the poll. You cannot end polls from other users.
+
+Returns a [message](/resources/message#message-object) object. Fires a [Message Update](/topics/gateway-events#message-update) Gateway event.
+
+<RouteHeader method="PUT" url="/channels/{channel.id}/polls/{message.id}/answers/@me">
+  Submit Poll Vote
+</RouteHeader>
+
+Submit [poll](#poll-structure) votes. Returns a 204 empty response on success. Fires multiple [Message Poll Vote Add](/topics/gateway-events#message-poll-vote-add)/[Message Poll Vote Remove](/topics/gateway-events#message-poll-vote-remove) Gateway events.
+
+###### JSON Params
+
+| Field      | Type          | Description                            |
+| ---------- | ------------- | -------------------------------------- |
+| answer_ids | array[string] | Selected answers, empty to clear votes |
```

Comment:
```suggestion
| answer_ids | array[integer] | Selected answers, empty to clear votes |
```

These are actually integers, the client just casts them into strings because reasons